pkg/network/config/config.go

```diff
@@ -265,6 +268,9 @@ type Config struct {
 	// EnableUSMQuantization enables endpoint quantization for USM programs
 	EnableUSMQuantization bool
 
+	// RingbuffersEnabled specifies whether ringbuffers are enabled or not
+	RingbuffersEnabled bool
```

Comment:
We have a similar value called EnableUSMRingBuffers
I think either we should rename this new configuration (`NPMRingbufferEnabled`) or to use the same value

cc @p-lambert

Reply:
since the configuration is under the `network_config` I think you should rename it to be npm only